### PR TITLE
Lesson-3

### PR DESCRIPTION
**Pull request by MaryiaTS**
wrote 3 functions. the first one check if the word is a palindrome. the second one compares numbers and outputs the biggest or the fewest. the last one replaces all 0 at the array with the string 'zero'.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,50 +1,135 @@
...
+// }
+//
+// //3rd task
+// const array3 = [2, 5, 8, 15, 0, 6, 20, 3];
+//
+// for (let i = 0; i < array3.length; i++) {
+//   if (array3[i] % 2 === 0) {
+//     console.log(array3[i]);
+//   }
+// }
```
Comment (5tronciy) [inline main.js]: Don't comment on the code for other lessons. Leave it in another branch and remove it here.

File: main.js [hunk 2]
```diff
@@ -1,50 +1,135 @@
...
+        newArray[i] = "zero";
+      }
+    } else {
+      newArray[i] = "1zerozero";
+    }
   }
-}
\ No newline at end of file
+  console.log(newArray);
+}
```
Comment (5tronciy) [inline main.js:79]: Make sure the function doesn't return undefined.

**Review by 5tronciy — APPROVED** [review]
The issues are not fatal and maintain requirements are met. So, I approve of your work.

File: main.js (hunk 2 above)
Comment (MaryiaTS) [reply main.js:79]: I checked the function, the required array is displayed
main.js:128 (10) [12, 53, '2zero', 18, 22, '1zerozero', 43, 57, '5zero', 1]
main.js:128 (10) [16, 38, '5zero', 'zero', 15, 97, 56, '3zero', 99, '9zero']

Comment (5tronciy) [reply main.js:79]: Every function returns smth. If the return statement is missed it returns the default undefined in JavaScript.

It's clear to return the array from this function and use the log out of the function as usual.

**Review by 5tronciy — APPROVED** [review]

**Pull request merged**